packages/core/components/DataSourcePrompt/index.tsx

```diff
@@ -23,6 +23,12 @@ const ADDITIONAL_COLUMN_DETAILS = [
     'If an "Uploaded" column is present, it should contain the date the file was uploaded to the storage and be formatted as YYYY-MM-DD HH:MM:SS.Z where Z is a timezone offset. ',
 ];
 
+export enum DataSourceType {
+    default = 0,
+    metadata = 1,
+    provenance = 2,
```

Comment:
Just added this in for now so that I could re-use the data source prompt to upload just a provenance file on its own